### ISSUE TITLE
Should Style/DateTime complain about to_datetime?

### ISSUE THREAD

**Issue by Ana06** [rubocop-hq/rubocop#6058]
I had a code like:

`Time.at(1409642056).to_datetime`

when changing it (to save the conversion) to:

`DateTime.strptime('1409642056', '%s')`

the `Style/DateTime` cop started complaining. I wonder if it should also complain in the first case, as both options are equivalent.

## RuboCop version

```
$ rubocop -V
0.57.2 (using Parser 2.5.1.0, running on ruby 2.5.0 x86_64-linux-gnu)
```
